doc/api.md

```diff
@@ -201,6 +340,179 @@ used should be taken from those defined by
+`IOTA_APPEND_MODE` environment variable (more info
+[here](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md)). If this flag is
+activated, the update requests to the Context Broker will be performed always with APPEND type, instead of the default
+UPDATE. This have implications in the use of attributes with Context Providers, so this flag should be used with care.
+
+#### Differences between `autoprovision`, `explicitAttrs` and `appendMode`
+
+Since those configuration parameters are quite similar, this section is intended to clarify the relation between them.
+
+If `autoprovision` is set to `true` (default case), the agent will perform an initial request creating a new entity into
+the Context Broker with **only** the static and active attributes provisioned in the config group, and also a new Device
+in the agent, every time a measure arrives with a new `device_id`. Otherwise, this measure is ignored. This is something
+related to the **southbound**.
+
+What `explicitAttrs` does is to filter from the southbound the parameters that are not explicitly defined in the device
+provision or config group. That also would avoid propagating the measures to the Context Broker.
+
+The default way the agent updates the information into the Context Broker is by using an update request. If
+`appendMode=true`, the IoTA will use an append request instead of an update one. This means it will store the attributes
+even if they are not present in the entity. This seems the same functionality that the one provided by `autoprovision`,
+but it is a different concept since the scope of this config is to setup how the IoT interacts with the context broker,
+this is something related to the **northbound**.
+
+Note that, even creating a group with `autoprovision=true` and `explicitAttrs=true`, if you do not provision previously
+the entity in the Context Broker (having all attributes to be updated), it would fail if `appendMode=false`. For further
+information check the issue [#1301](https://github.com/telefonicaid/iotagent-node-lib/issues/1301).
+
+### Timestamp Compression
+
+This functionality changes all the timestamp attributes found in the entity, and all the timestamp metadata found in any
+attribute, from the basic complete calendar timestamp of the ISO8601 (e.g.: 20071103T131805) to the extended complete
+calendar timestamp (e.g.: +002007-11-03T13:18). The middleware expects to receive the basic format in updates and return
+it in queries (and viceversa, receive the extended one in queries and return it in updates).
+
+### Timestamp Processing
+
+The IOTA processes the entity attributes looking for a `TimeInstant` attribute. If one is found, for NGSI v2, the plugin
+adds a `TimeInstant` attribute as metadata for every other attribute in the same request. With NGSI-LD, the Standard
+`observedAt` property-of-a-property is used instead.
+
```

Comment:
This is now core functionality. (before they were plugins). Added as functionality in topics section